src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2066,6 +2066,8 @@ image::upload-catalog-objects.png[align=left]
 
 * *Content Type*: The Content Type is used to indicate the media type of the resource. New content Types can be added by clicking the `plus` button.
 
+* *Project Name*: The name of the project to which belongs the object with type no workflow. We assume that every workflow has project name.
```

Comment:
The name of the project to which belongs the object. The project name can be provided for all objects except workflows. We assume that the project name is defined in the workflow XML file.